src/coreclr/jit/regalloc.cpp

```diff
@@ -320,13 +320,6 @@ void Compiler::raMarkStkVars()
                 needSlot |= varDsc->IsAddressExposed();
             }
 
-#if FEATURE_FIXED_OUT_ARGS
-
-            /* Is this the dummy variable representing GT_LCLBLK ? */
-            needSlot |= (lclNum == lvaOutgoingArgSpaceVar);
-
-#endif // FEATURE_FIXED_OUT_ARGS
-
```

Comment:
This is no longer necessary since `lvaOutgoingArgSpaceVar` is a normal address-exposed struct local now and handled above.